config/config.json

```diff
@@ -1,9 +1,20 @@
 {
-"settings":[
-		{"topic":"test", "scripts":[
+"transformers_settings":[
```

Comment:
Can you please open an issue where do you describe the structure and the fields of this configuration file? Thank you